doc/ref_model/chapters/chapter04.md

```diff
@@ -181,7 +181,7 @@ This section covers a list of implicit NFVI capabilities and measurements that d
 | Ref | NFVI capability | Unit | Definition/Notes |
 |--------------------|------------------------------------------|--------|---------------------------------------------------------------------------------------------------------------------|
 | i.nfvi.cap.016 | CPU overbooking | 1:N | <definition update scheduled> |
-| i.nfvi.cap.017 | vNIC QoS | Yes/No | QoS enablement |
+| i.nfvi.cap.017 | vNIC QoS | Yes/No | QoS enablement <br> **Note:** This capability is not applicable for containers due to the lack of vNIC, however VLAN QoS or DSCP marking can be implemented.|
```

Comment:
@CsatariGergely Does this capability need to be identified as VNF-specific and treated appropriately? Would changing it to virtual _interface_ make it more applicable to containers? If not, do you envision that at some point in the future, this object/concept will become applicable to containers? Is its lack of applicability, CNI-dependent (i.e. implementation dependent)?

Reply:
In case of containers there are only interfaces and not vNIC-s.

Reply:
I like the idea of changing the wording to be agnostic. Another example is bare metal, where a vNIC or virtual interface might not be an appropriate term?

Reply:
ETSI Network Functions Virtualisation (NFV);Management and Orchestration GS NFV-MAN 001 V1.1.1 (2014-12) defines connection point:
connection point: information element representing the virtual and/or physical interface that offers the network connections between instances of NS, VNF, VNFC (based on the VDU (Virtualisation Deployment Unit) information element), PNF and a VL (Virtual Link)
NOTE: Some examples of virtual and/or physical interfaces are a virtual port, a virtual NIC address, a physical port, a physical NIC address or the endpoint of an IP VPN.